finance/funding_requests.md

```diff
@@ -0,0 +1,52 @@
+
+Funding requests can be made to the Astropy Finance Committee.  The following provides a description for how to obtain funding for different types of activities. 
+
+
+## Travel
+
+1. Send an email to astropy-finance-committee@googlegroups.com with the following information:
+* Name
+* Description
+* Total Request
+* Itemized Budget
+
+Costs for per diem can use the [US Government per deim rates | https://www.gsa.gov/travel/plan-book/per-diem-rates], and airfare costs should be justified. 
```

Comment:
"should be justified" is a bit ambiguous... What counts as justification?  Although perhaps this really requires a clear airfare policy (wich is pending) to really be clear?